frontend/src/app/main/ui/viewer/interactions.cljs

```diff
@@ -84,7 +84,14 @@
 
         ;; Retrieve frames again with correct modifier
         frame   (get objects-not-fixed (:id frame))
-        base    (get objects-not-fixed (:id base))]
+        base    (get objects-not-fixed (:id base))
+
+        non-delay-interactions (->> (:interactions frame)
+                                    (filter #(not= (:event-type %) :after-delay)))
```

Comment:
interactions, should be a vector, but you are converting it to seq, `filterv` should be used